deconnable_everything.md

```diff
@@ -0,0 +1,107 @@
+
+## Abstract
+Dozens of snowflake and legacy objects are straight up indestructible in CM through normal means. Most people who play CM regulary grown used to it but it is 
+shockingly apparent when you actually to try to notice it or when someone abuses it. Things like filing cabinets, hydroponics trays ore boxes, storage tanks, showcases 
+and computers are indestructible through normal means (if at all) for god-knows-why and are constantly used for cheesing every now and then. If the CLF threw away their armor and
+just wore crates (100% bulletproof), they would control the galaxy already.
+
+Furthermore, many things also lack proper deconstruction. You should be able to deconstruct nearly everything destructible ( within balance bounds, of course)
+
+***Disclaimer: UNINTENTIONAL objects meaning things that aren't meant to be indestructable or affect balance is a big way if its destructable, like hulls ***
+<!-- An abstract is a short blurb, about a paragraph or two, succinctly describing your feature. This should mostly be "why", but can include "what". -->
+
+## Goals
+
+1. Find nearly all ***unintentional*** indestructable things into being destructable and deconstructable
+3. Give nearly all of the above deconstruction paths.
+  - Deconstructable objects should give a reasonable amount of mats depending on the rarity of the obj and how complex the deconstruction path is. 
+  Exotic machinery found in atmos or the handful of generators should gain exotic materials logically.
+  - ex: Big Scrubbers yielding plasteel after some time of deconstructing
+ 
+ Deconning objects should ALWAYS give more rewards than simply destroying
+
+3. Give some of the more used objects actual recipes to build them back again
+  - If you are able to deconstruct it, you should be able to build it again
+  - Things like hydroponics trays that are commonly used so entire roles wont get screwed because it got deconstructed.
+  - Not everything should have it, of course. There's already enough snowflake objects, we dont need more.
+
+4. Give crafting recipes to stuff that SHOULD have it but do not
+  - Things like wooden tables, pool tables, carpets,  etc. Why on earth isnt there an easy way to make them. MT buff.
+<!-- This is a numbered list clearly detailing your goals for the feature. As per usual, this should be a mixture of both why and what. -->
+
+## Non-goals
+
+1. Nothing should be affecting balance in an adverse way
+  - We obviously don't want people getting entire stacks of plasteel from 1 area.
+  - This is not a major balance affecting MR and shouldn't be.
+
+2. Intentionally indestructable objects should not be touched
+  - this has been repeated throughout this doc but putting it here so someone won't say *"SUPER MARINE/XENO BUFF THEY CAN DESTROY HULL WALLS!!!111"*
+<!-- Just like goals, but the opposite! Every feature has boundaries it won't step over. These should be written as if they start with "We will not...". -->
```

Comment:
Again, this is including implementation detail, it should only include design detail.